gremlin-client/src/structure/value.rs

```diff
@@ -333,6 +334,36 @@ impl std::convert::TryFrom<GValue> for i64 {
     }
 }
 
+impl std::convert::TryFrom<GValue> for f32 {
```

Comment:
Just a heads up this is a duplicate implementation of the same code in https://github.com/wolf4ood/gremlin-rs/pull/129/files to keep the PRs independent for review and merging. One of the implementations will need to be removed.